helper/helper.go

```diff
@@ -245,29 +217,6 @@ func getOidListFromFile(oidFileName string) ([]int, error) {
 	return oidList, nil
 }
 
-func flushAndCloseRestoreWriter(pipeName string, oid int) error {
```

Comment:
why we need to move and rename this function? If this change is reverted here and in DoCleanup (and with name update inside the `func (RestoreHelper) flushAndCloseRestoreWriter(pipeName string, oid int)`), everything seems to works fine. It will reduce the delta significantly.

Reply:
Because this function is only valid for restore.

Reply:
> And without moving how can it be mocked in tests?

We're mocking the `func (h *restoreMockHelperImpl) flushAndCloseRestoreWriter(pipeName string, oid int)` in the test. I see no problem here. The helper test works fine on my side with this change.

> Because this function is only valid for restore

It is currently called from the common `DoCleanup()` in helper.go

Reply:
> > Because this function is only valid for restore
> 
> It is currently called from the common `DoCleanup()` in helper.go

Yes but `writeHandle` and `writer` exist only in restore

Reply:
Even function name suggests that it's for restore: flushAndClose**Restore**Writer()

Reply:
Then why not to remove it from the common DoCleanup() and place in the restore's DoCleanup?